src/main/java/uk/gov/hmcts/reform/prl/mapper/citizen/CaseDataApplicantElementsMapper.java

```diff
@@ -73,6 +74,7 @@ private static PartyDetails buildPartyDetails(ApplicantDto applicantDto) {
                 .isAddressConfidential(buildConfidentialField(contactDetailsPrivateList, ADDRESS_FIELD))
                 .isEmailAddressConfidential(buildConfidentialField(contactDetailsPrivateList, EMAIL_FIELD))
                 .isPhoneNumberConfidential(buildConfidentialField(contactDetailsPrivateList, TELEPHONE_FIELD))
+                .doTheyHaveLegalRepresentation(YesNoDontKnow.no)
```

Comment:
relevant?

Reply:
yes, it's citizen to applicant mappers.
Just added LegalPresentation to No for citizen applicants.